src/openforms/contrib/kvk/tests/test_validators.py

```diff
@@ -119,3 +123,107 @@ def test_branchNumber_validator(self):
             ValidationError, _("Expected a numerical value.")
         ):
             validator("bork")
+
+
+class KvKRemoteValidatorCachingTestCase(KVKTestMixin, SimpleTestCase):
```

Comment:
@stevenbal I ran into test failures with VCR in either of two modes:

* Re-recording the cassettes - the `play_count` is always set to zero because it's still building the cassette
* Replaying the cassettess - the `self.cassette.requests` size reports all the known interactions, and is fine when re-recording the cassettes because it's still building up the history.

There's no obvious way to make checks on which requests were made and how often with VCR, so instead I opted for requests-mock here with a dummy response to check the number of outgoing requests made at any given point.

Reply:
Hmm alright, I wasn't sure about the VCR approach for the caching assertions either, so this is probably the best way to fix it

Reply:
yeah this didn't feel very "exact" when to use which approach to me either